workspace-manager-clienttests/src/main/java/scripts/utils/WorkspaceManagerServiceUtils.java

```diff
@@ -99,4 +111,14 @@ private static ApiClient buildClient(AccessToken accessToken, ServerSpecificatio
 
         return apiClient;
     }
+
+    public static void assertHttpOk(WorkspaceApi workspaceApi, String label) {
+        int httpCode = workspaceApi.getApiClient().getStatusCode();
+        logger.info("{}} HTTP code: {}", label, httpCode);
+        // Code will initialize to zero for non-overridden methods like setup()
+        if (httpCode > 0) {
+            assertThat(httpCode, greaterThanOrEqualTo(HttpStatusCodes.STATUS_CODE_OK));
+            assertThat(httpCode, lessThan(300));
```

Comment:
how about `HttpStatusCodes.isSuccess(httpCode)`?

Reply:
love it